mmdet3d/models/backbones/mink_resnet.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Follow https://github.com/NVIDIA/MinkowskiEngine/blob/master/examples/resnet.py # noqa
+# and mmcv.cnn.ResNet
+try:
+    import MinkowskiEngine as ME
+    from MinkowskiEngine.modules.resnet_block import BasicBlock, Bottleneck
+except ImportError:
+    import warnings
+    warnings.warn(
+        'Please follow `getting_started.md` to install MinkowskiEngine.`')
+    # blocks are used in the static part of MinkResNet
+    BasicBlock, Bottleneck = None, None
+
+import torch.nn as nn
+
+from mmdet.models import BACKBONES
+
+
+@BACKBONES.register_module()
+class MinkResNet(nn.Module):
+    """Minkowski ResNet backbone. See 4D Spatio-Temporal ConvNets
+    (https://arxiv.org/abs/1904.08755) for more details.
```

Comment:
Please follow this [line](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/detectors/voxelnet.py#L15) for the reference format in the docstring. It can be formatted as a hyper-link in the documentation.